src/routes/OrganizationsList/OrganizationsList.tsx

```diff
@@ -448,12 +463,40 @@ function PageContent() {
                 <Td dataLabel={columnNames.name}>
                   <Link to={org.name}>{org.name}</Link>
                 </Td>
-                <Td dataLabel={columnNames.repoCount}>{org.repoCount}</Td>
-                <Td dataLabel={columnNames.teamsCount}>{org.teamsCount}</Td>
-                <Td dataLabel={columnNames.membersCount}>{org.membersCount}</Td>
-                <Td dataLabel={columnNames.robotsCount}>{org.robotsCount}</Td>
+                <Td dataLabel={columnNames.repoCount}>
+                  {org.repoCount !== null ? (
+                    org.repoCount
+                  ) : (
+                    <Skeleton width="100%" />
+                  )}
+                </Td>
+                <Td dataLabel={columnNames.teamsCount}>
+                  {org.teamsCount !== null ? (
+                    org.teamsCount
+                  ) : (
+                    <Skeleton width="100%" />
+                  )}
+                </Td>
+                <Td dataLabel={columnNames.membersCount}>
+                  {org.membersCount !== null ? (
+                    org.membersCount
+                  ) : (
+                    <Skeleton width="100%" />
+                  )}
+                </Td>
+                <Td dataLabel={columnNames.robotsCount}>
+                  {org.robotsCount !== null ? (
+                    org.robotsCount
+                  ) : (
+                    <Skeleton width="100%" />
+                  )}
+                </Td>
                 <Td dataLabel={columnNames.lastModified}>
-                  {formatDate(org.lastModified)}
+                  {org.lastModified !== null ? (
```

Comment:
Not sure if we want to conflate the state of loading with the loaded value of `null`. eg. I created a new organization that has `last_modified: null` so it show's a skeleton that won't go away.

Given this may go away entirely with the changes we discussed

Reply:
Hmm, we use `null` in other parts of the code to check state for skeletons so we can differentiate between loaded, requested and not found. Does `lastModified` default to `null` if not found?

Reply:
That's correct.